core/pkg/net/ssl/ssl.go

```diff
@@ -125,6 +127,56 @@ func AddOrUpdateCertAndKey(name string, cert, key, ca []byte) (*ingress.SSLCert,
 	}, nil
 }
 
+// AddOrUpdateCertAuth creates a .pem file with the specified CAs to be used in Cert Authentication
+func AddOrUpdateCertAuth(name string, ca []byte) (*ingress.SSLCert, error) {
+
+	caName := fmt.Sprintf("ca-%v.pem", name)
+	caFileName := fmt.Sprintf("%v/%v", ingress.DefaultSSLDirectory, caName)
+
+	tempCAPemFile, err := ioutil.TempFile(ingress.DefaultSSLDirectory, caName)
+	glog.V(3).Infof("AddOrUpdateCertAuth: Creating temp file %v for CA: %v", tempCAPemFile.Name(), caName)
```

Comment:
Yes, you're right. This was because of some debugging, forgot it here.

Also moved the line as requested.